com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -918,11 +993,11 @@ void SendInfoToBrain()
             {
                 if (m_PolicyFactory.BrainParameters.VectorActionSpaceType == SpaceType.Discrete)
                 {
-                    CollectDiscreteActionMasks(m_ActionMasker);
+                    m_Actuators.WriteActionMask();
```

Comment:
Not sure we need to it just yet, but we might want to remove the "if(spaceType == Discrete)" around this and make sure it gets handled in WriteActionMask instead.

Reply:
Probably a good sanity check is to see where else in Agent we check the space type for anything. It will probably either need to go away now, or at least soon.

Reply:
makes sense